data/dbus/yggd.conf.in

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE busconfig PUBLIC "-//freedesktop//DTD D-BUS Bus Configuration 1.0//EN" "https://dbus.freedesktop.org/doc/busconfig.dtd">
+<busconfig>
+  <policy user="@user@">
```

Comment:
Following line is missing:
```
<allow own="com.redhat.Yggdrasil1"/>
```
Otherwise, it is not possible to start the service.

Reply:
Oh interesting! I had not considered whether the order in which lines appear matters.

Reply:
I switched the order of these two `<allow>` tags.